bin-dir/system-test-script.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+cd ..
+
+echo "starting collection roller"
+bin/collection-roller &> /dev/null &
```

Comment:
We should be writing these logs to file for debugging instead of writing them to /dev/null

Reply:
changed and writing them to logfile file.